.travis.yml

```diff
@@ -8,22 +8,25 @@ before_install:
   - gem update --system
   - gem update bundler
 rvm:
-  - jruby-9.1.9.0
+  - jruby-9.1
   - jruby-9000
   - 1.9.3-p551
   - 2.0.0-p648
-  - 2.1.10
-  - 2.2.9
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - 2.1
```

Comment:
Not specifying the patch number here installs the latest patch.

Reply:
was travis breaking down because of outdated patches?